### ISSUE TITLE
询问下batch_size

### ISSUE THREAD

**Issue by tzbs** [Zc0812/DecoupledESC#1]
前辈，您好，我想问下，您这篇论文里sft阶段用的batch_size为32，是梯度累积的吗，您用了4张卡，如果每张卡的batch设置为8，会爆显存呀？max_token_length是多少呢？3072可以吗？ESConv数据集（训练+验证=1040+130）
[INFO|2025-11-14 17:15:48] trainer.py:2519 >> ***** Running training *****
[INFO|2025-11-14 17:15:48] trainer.py:2520 >>   Num examples = 14,743
[INFO|2025-11-14 17:15:48] trainer.py:2521 >>   Num Epochs = 3
[INFO|2025-11-14 17:15:48] trainer.py:2522 >>   Instantaneous batch size per device = 1
[INFO|2025-11-14 17:15:48] trainer.py:2525 >>   Total train batch size (w. parallel, distributed & accumulation) = 32
[INFO|2025-11-14 17:15:48] trainer.py:2526 >>   Gradient Accumulation steps = 8
[INFO|2025-11-14 17:15:48] trainer.py:2527 >>   Total optimization steps = 1,383
谢谢您嘞！
